### ISSUE TITLE
File not found issue in notebook 3

### ISSUE THREAD

**Issue by lawlietsoul** [anthony-wang/BestPractices#2]
Dear Wang,

thank for this great tutorial!
In 3-modeling_classic_models.ipynb by excuting generate_features() I got a FileNotFoundError as it told me 'liynyk.csv' is not found, I think the elem_prob is defined as 
+ '/cbfv/element_properties/'
+ elem_prop
+ '.csv')
so changing folder name CBFV to cbfv would be a temporary solution.

**Comment by anthony-wang** [comment]
Hey @lawlietsoul, thanks for reaching out, and thanks!

This seems to be an issue with relative paths.
Could you please let me know how you are starting the Jupyter notebook? For me, it works when I launch the Jupyter kernel in the root directory (i.e. directly launching `jupyter notebook` in the `BestPractices/` root directory, and not in `BestPractices/notebooks/`).

Could you try that and let me know if that fixes the issue?

**Comment by lawlietsoul** [comment]
Hello Wang, 
thanks for the reply. I also tried to start jupyter directly from /BestPractices instead of /BestPractices/notebooks, after lauching the jupyter I come into notebooks to start practice3, and print it inside the notebook it tells me I am in the /BestPractices/notebooks.  But I still need to rename CBFV to cbfv to make things work. I also agreed it should be caused by the relative paths, mayebe I did something else wrong. Thanks for the reply!
